region/maridia/outer/Mama Turtle Room.json

```diff
@@ -397,11 +397,43 @@
         ]}
       ]
     },
+    {
+      "link": [1, 3],
+      "name": "Ride Mama Turtle on the Right (Grab Item)",
+      "requires": [
+        "h_canNavigateUnderwater",
+        "canUseEnemies",
+        "canCarefulJump",
+        {"or": [
+          "canTrickyDodgeEnemies",
+          {"and": [
+            "Gravity",
+            "canTrickyJump"
+          ]},
+          {"enemyDamage": {
+            "enemy": "Kame (Tatori)",
+            "type": "contact",
+            "hits": 1
+          }}
+        ]}
+      ],
+      "clearsObstacles": ["A"],
+      "note": [
+        "Requires getting back on Mama Turtle while she is at the right.",
+        "The safest way is to have a normalized fall speed with either a full height jump or falling from the ledge above.",
+        "If suitless, walking off the right ledge when she hits the left wall can be a good indicator to help land on her safely.",
+        "Shrinking Samus' hitbox after touching the turtles back will also usually help.",
+        "While near the right wall, shoot the item and grab it. If the shot misses, carefully fall back down and safely land on Mama Turtle again.",
+        "After grabbing the item, fall on her again to avoid taking a hit; jump to the left at the top of the ride to escape the water safely."
+      ],
+      "devNote": "FIXME: Is it worth adding a suitless bootless strat that follows this and gets back to the left without another hit?"
```

Comment:
This strat does mean that you can ride mama turtle again for free after collecting the item.  Either to reach the door safely or to jump to 6.